app/app.go

```diff
@@ -732,17 +732,28 @@ func getNetworkInfo(app *App, ctx sdk.Context, manifest *UpgradeManifest) (*Netw
 	var networkInfo *NetworkConfig
 	var err error
 	if app.cudosMigrationConfigPath != "" {
-		app.Logger().Info("Loading network config", "file", app.cudosMigrationConfigPath, "hash", app.cudosMigrationConfigSha256)
+		app.Logger().Info("cudos merge: loading network config", "file", app.cudosMigrationConfigPath, "hash", app.cudosMigrationConfigSha256)
```

Comment:
(this comment belongs to line [730](https://github.com/fetchai/fetchd/pull/383/files/959f5dd25a6f9ef8fd5a512405ee79c400879fe0..615a29b4b604b9aecbbff8ceae440357058b5251#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R730) - github UI didnlt allow me to put it there)
We would need to add new parameter either for the input genesis  content, or for `mergeSourceChainId string`, in order to pass in the Merge Source ChainID value in for the check:
```go
func getNetworkInfo(app *App, ctx sdk.Context, mergeSourceGenesisAppStateJson *map[string]interface{}, manifest *UpgradeManifest) (*NetworkConfig, error) {
```